templates/site/index.html

```diff
@@ -16,8 +16,7 @@
     {% with %}
         {% set title = "About Marble" %}
         {% set content = "<p>Marble is a network of cloud based web services <a class='paragraph-href' href='node.html'>nodes</a>
-        that provide access to large climate and satellite
-        <a class='paragraph-href' href='https://redoak.cs.toronto.edu/stac-browser/'>data sets</a>.
+        that provide access to large climate and satellite data sets.
```

Comment:
I'll let Shruti weigh in on this since this link is in her design.  I think the decision was a link to something was better than no link.

Reply:
I'm interested in @ShrutiK100 's opinion as well. My opinion is that a link that is misleading is worse than no link. 

This text implies that the data on the network can be found at that link. Only a small portion of the data can be found there. If that changes in the future, we can add a link back in.

Reply:
Makes sense to remove the link